src/Installer/redist-installer/targets/GenerateBundledVersions.targets

```diff
@@ -590,6 +590,10 @@ Copyright (c) .NET Foundation. All rights reserved.
                              TargetFramework="net10.0"
                              WebAssemblySdkPackVersion="$(MicrosoftNETCoreAppRuntimePackageVersion)" />
 
+    <KnownAspNetCorePack Include="Microsoft.AspNetCore.App.Internal.Assets"
+                             TargetFramework="net10.0"
+                             AspNetCorePackVersion="$(MicrosoftAspNetCoreAppRuntimePackageVersion)" />
+
```

Comment:
This is the only part I'm not 100% sure about. You can have an SDK and multiple runtimes. Meaning that you can have an  install that has 1.0.0, 1.0.1, and 1.0.2 installed. 1.0.2 might be bundled, but your app might be using `<FrameworkReference="Microsoft.AspNetCore.App" Version="1.0.0" />` or have it pinned in the `global.json`. 

In that case shouldn't we respect that?

@maraf do you have any thoughts?

Reply:
Similar question. Do we have a way for this to only apply to 10+?

Reply:
@maraf but it depends on whether you are pinning explicitly to a previous runtime, isn't it?